Documentation/subcommands/config.md

```diff
@@ -0,0 +1,125 @@
+# rkt config
+
+This command prints the rkt configuration per stage in JSON format.
```

Comment:
The `config` subcommand prints the configuration of each rkt stage in JSON on the standard output.